ecpy/tasks/tasks/util/formula_task.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015 by Ecpy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Task for defining mathematical formulas.
+
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+
+from collections import OrderedDict
+from traceback import format_exc
+from ast import literal_eval
+
+from atom.api import (Typed, set_default)
+
+from ....utils.atom_util import (ordered_dict_from_pref, ordered_dict_to_pref)
+
+from ...base_tasks import SimpleTask
+
+
+class FormulaTask(SimpleTask):
+    """Compute values according to formulas. Any valid python expression can be
+    evaluated and replacement to access to the database data can be used.
```

Comment:
First sentence can remain on first line next ones should be separated from the first by a blank line (see Numpy docstring formatting guidelines).